packages/scss/src/components/inputs/textfield/_textfield.compact.scss

```diff
@@ -58,13 +35,8 @@
 	}
 
 	.textfield-options {
-		left: auto;
-		margin: 0;
-		min-width: 0;
-		overflow-x: hidden;
-		overflow-y: auto;
 		right: 0;
-		top: 100%;
+		left: auto;
 		width: calc(100% - #{_component("textfield.compact.label-right-margin")} - #{_component("textfield.compact.label-sizes.default")});
 	}
 
```

Comment:
`.texttfield-suffix` shouldn't need to be defined there if it is properly defined in the parent class.

Reply:
✔️